Karma/Models/Post.cs

```diff
@@ -25,7 +25,7 @@ public class Post : IComparable<Post>
         [MaxLength(30)]
         public string Title { get; set; }
         [Required]
-        public Enums.Category ItemType { get; set; }
+        public int ItemType { get; set; }
```

Comment:
I tried to create a post and not select an itemtype (left the Category field with the selection 'Select...'). Currently there's no error if you don't select an itemtype, the default value is just '0', which is an actual itemtype.

Reply:
I'm really thankful you noticed this early on. I'm going to fix this immediately.

Reply:
#https://github.com/PauliusLu/Pro-Bono/pull/22/commits/f2bd88d4d9ec7db5a1974d203f11e3ec04850b7b